homeassistant/components/sensibo/climate.py

```diff
@@ -57,6 +57,7 @@
 AVAILABLE_FAN_MODES = {
     "quiet",
     "low",
+    "medium_low",
```

Comment:
I think we need to add a translationkey for this one

Reply:
Thanks!